src/organizations/components/OrgProfileTab/SuspendPaidOrgOverlay.tsx

```diff
@@ -158,6 +160,16 @@ export const SuspendPaidOrgOverlay: FC = () => {
         >
           You will be able to recover this organization's data for up to 7 days
           by contacting support. It will be unrecoverable afterwards.
+          {params.users > 0 && (
+            <>
```

Comment:
Same here: I might add a guard `params?.users` so that we don't get a `ReferenceError` if the users param is not passed to the overlay controller.

Reply:
I added a fallback value for userCount `userCount: users.length || 0` so we are sure to get a params value in this component